src/containers/Order/index.tsx

```diff
@@ -0,0 +1,73 @@
+import styles from './styles.module.scss';
+import classnames from 'classnames';
+import { TO_ORDERS } from '@/configs';
+import { Paper, Grid, Group } from '@mantine/core';
+import OrderTable from '../../components/OrderTable';
+import ListPageLayout from '@/layout/SubPageLayout';
+export default function Order() {
+  return (
+    <Paper shadow="xs" radius="md" p="md">
+      <ListPageLayout rootDir={TO_ORDERS} title="Admin Order List">
+        <div className={classnames(styles.Card, styles.mb_4)}>
```

Comment:
Because `mb_4` only receive `margin-bottom` property. This can be delete and move that property to `styles.card`.

Note: wrong naming convention. Should be `card` instead of `Card`

Reply:
done